### PR TITLE
Update default entity properties in Create

### PR DESCRIPTION
**Pull request by huffman**
https://highfidelity.fogbugz.com/f/cases/19040/Audit-default-values-for-properties

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13923/HighFidelity-Beta-PR14214-7b60505.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13923/HighFidelity-Beta-PR14214-7b60505.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13923/HighFidelity-Beta-Interface-PR14214-7b60505.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13923/HighFidelity-Beta-PR14214-7b60505.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13923/HighFidelity-Beta-Interface-PR14214-7b60505.dmg

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13930/HighFidelity-Beta-PR14214-738ac30.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13930/HighFidelity-Beta-PR14214-738ac30.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13930/HighFidelity-Beta-Interface-PR14214-738ac30.dmg

File: scripts/system/edit.js [hunk 1]
```diff
@@ -294,6 +297,187 @@ function checkEditPermissionsAndUpdate() {
...
+        shapeType: "box",
+        collisionless: true,
+        modelURL: IMAGE_MODEL,
+        textures: JSON.stringify({ "tex.picture": DEFAULT_IMAGE })
+    },
+    Web: {
+        sourceUrl: "https://highfidelity.com/",
+        dpi: 30,
+    },
+    Particles: {
```
Comment (thoys) [inline scripts/system/edit.js]: Shouldn't this be ParticleEffect ?

Comment (huffman) [reply scripts/system/edit.js]: Good catch. I thought it was working, must not have been.

**Comment by huffman** [comment]
# Test Plan

  1. Go to your domain
  2. Enable create
  3. Ensure that all entity types can be created from the Create window
  4. Ensure that the entity properties of each type match those in the spreadsheet shared with you

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/14146/HighFidelity-Beta-PR14214-a4adf2c.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/14146/HighFidelity-Beta-Interface-PR14214-a4adf2c.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/14146/HighFidelity-Beta-PR14214-a4adf2c.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/14146/HighFidelity-Beta-Interface-PR14214-a4adf2c.dmg

**Comment by huffman** [comment]
@shanzzam Updated with the manuscript ticket, thanks!

**Comment by qligmaloney** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/14181/HighFidelity-Beta-PR14214-a4adf2c.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/14181/HighFidelity-Beta-PR14214-a4adf2c.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/14181/HighFidelity-Beta-Interface-PR14214-a4adf2c.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/14181/HighFidelity-Beta-PR14214-a4adf2c.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/14181/HighFidelity-Beta-Interface-PR14214-a4adf2c.exe

**Comment by QliCknox** [comment]
Currently Testing

**Comment by QliCknox** [comment]
Stopped testing

End of day, got through over half of the PR and will be checking bugs against master/stable beginning of Monday work day

**Comment by QliCknox** [comment]
Resumed Testing

**Comment by QliCknox** [comment]
FAIL @conklin94122 

Shapes: There is no area for a MaterialURL in Shapes. This is not visible in Master either.
Zone: Keylight defaults to -90.00 instead of 45 like the spreadsheet expects. Keylight defaults to 90.00 in Master 9495.
Zone: Haze Colour defaults to #809a81. This does not occur in Master
Zone: Glare Colour defaults to #ffe5b3. This does not occur in Master
Model: When the user makes a Model, the URL is cleared. This does not occur in Master.
Model: Spreadsheet expects 'Textures' box in Model to be {}, and Original Texture to be clear. This is flipped. This does occur in Master.
Image: ImageURL is not NULL. This does occur in Master.
Particles: Polar Start ( Vertical Angle) is 0 instead of 180. Occurs in Master.
Particles: Colour/Colour Start is #c8c8c8 instead of #ffffff
Light: Light Colour is #ffff00 instead of #ffffff. Occurs in master, though the light colour is #969696
Light: Spotlight is enabled by default. Does not occur on Master.

General: Dimensions aren't global
Global Scale isn't dimensional, its percentage based
Collides is false by default, though I'm curious if thats correct as the Spreadsheet asks for "Collides" to be false, but the property name is "Collisionless" and its disabled by default, so objects come in with collision.
Objects still come in as Grabbable by default
Lifetime is -1 (objects stick around) instead of 300

I noticed some visual bugs as well, but as for default properties, these are the areas found on both PC and Mac that do not match the supplied spreadsheet. Some occur on Master but I assumed it would be important to include them as properties are being changed in this PR from their current default or bugged values.

[Archive.zip](https://github.com/highfidelity/hifi/files/2502310/Archive.zip)

**Comment by huffman** [comment]
@QliCknox I went through and addressed the discrepancies. Some notes:

> Shapes: There is no area for a MaterialURL in Shapes. This is not visible in Master either.

This is OK

> Model: Spreadsheet expects 'Textures' box in Model to be {}, and Original Texture to be clear. This is flipped. This does occur in Master.

This is OK

> Particles: Colour/Colour Start is #c8c8c8 instead of #FFFFFF

This was an issue from a separate PR, it should now be fixed in master and in this PR.

> Objects still come in as Grabbable by default

This should depend on the setting set in the Edit menu

> Lifetime is -1 (objects stick around) instead of 300

This is OK

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/14276/HighFidelity-Beta-PR14214-d7dce45.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/14276/HighFidelity-Beta-PR14214-d7dce45.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/14276/HighFidelity-Beta-Interface-PR14214-d7dce45.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/14276/HighFidelity-Beta-PR14214-d7dce45.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/14276/HighFidelity-Beta-Interface-PR14214-d7dce45.exe

**Comment by qligmaloney** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/14350/HighFidelity-Beta-PR14214-d7dce45.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/14350/HighFidelity-Beta-PR14214-d7dce45.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/14350/HighFidelity-Beta-Interface-PR14214-d7dce45.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/14350/HighFidelity-Beta-PR14214-d7dce45.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/14350/HighFidelity-Beta-Interface-PR14214-d7dce45.exe

**Comment by QliCknox** [comment]
Currently Testing

**Comment by QliCknox** [comment]
Pass @conklin94122 

Properties are coming in at the expected defaults

**Pull request merged**